src/hooks/useVaultWithoutBalance/useVaultWithoutBalance.ts

```diff
@@ -0,0 +1,71 @@
+import { useYearn } from 'features/defi/contexts/YearnProvider/YearnProvider'
```

Comment:
I created a specific hook to fetch vaults with less data (precisely, I am not fetching pricePerShare as it is costly to do it for 70+ vaults at once) like it is done in `useVaultBalances.tsx`